styles.css

```diff
@@ -0,0 +1,90 @@
+body {
+    font-size: 16px;
+    background-image: linear-gradient(to top, #cfd9df 0%, #e2ebf0 100%);
+    margin: 0;
+    font-family: 'Roboto', sans-serif;
+    
+}
+
+.container {
+    background-color: gray;
+    position: relative;
+    margin: 200px;
+    display: flex;
+    flex-direction: column;
+    border-radius: 20px;
+    min-width: 500px;
+    box-shadow: 0px 0px 20px 0px rgba(0, 0, 0, 0.4);
+}
+
+.container__title {
+    background-color: #764ba2;
+    border-radius: 25px 25px 0px 0px;
+    width: 100%;
+    display: flex;
+    margin: auto;
+    color: white;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+}
+
+.container__title {
+    font-weight: 500;
+
+}
+
+.container__content{
+    background-image: linear-gradient(0deg, #667eea 0%, #764ba2 100%);
+    border-radius: 0px 0px 25px 25px;
+    padding: 20px;
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+
+.container__content-item{
+    position: relative;
+    background-color: rgba(255, 255, 255, 0.1);
+    width: 300px;
+    min-width: 300px;
+    border-radius: 20px;
+    border: 2px solid white;
+    color: white;
+    margin: 20px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    box-shadow: 0px 0px 5px 0px rgba(0, 0, 0, 0.4);
+}
+.container__content-item--title{
+    font-size: 30px;
+    margin-top: 40px;
+}
+
+.container__content-item--details{
+    margin: 15px 25px;
+}
+.container__content-item--checkbox{
+    display: flex;
+    align-items: center;
+    margin: 25px;
+    font-size: 20px;
+    
+}
+
+input[type='checkbox']{
```

Comment:
Don't use tag selector, use the classes instead of `input`

Reply:
Is easier to use input instead of tag